node/trusted_validators_membership_change.re

```diff
@@ -0,0 +1,45 @@
+open Helpers;
+open Protocol;
+open Mirage_crypto_ec;
+
+module type T = {
+  [@deriving yojson]
+  type action =
+    | Add
+    | Remove;
+  [@deriving yojson]
+  type t = {
+    action,
+    address: Address.t,
+  };
+  let read: (~file: string) => Lwt.t(list(t));
+  let write: (list(t), ~file: string) => Lwt.t(unit);
+};
+
+[@deriving yojson]
+type action =
+  | Add
+  | Remove;
+[@deriving yojson]
+type t = {
+  action,
+  address: Address.t,
+};
+let read =
+  read_json(json => {
+    let.ok validators = [%of_yojson: list(t)](json);
+    Ok(validators);
+  });
+let write = write_json(validators => validators |> [%to_yojson: list(t)]);
+
+let compare = (x, y) =>
+  Cstruct.compare(
+    Ed25519.pub_to_cstruct(x.address),
+    Ed25519.pub_to_cstruct(y.address),
+  );
```

Comment:
This compare is not right, what if both have the same address and different action?

Reply:
Doing in fieldwise compare in the next incoming commit